### PR TITLE
feat(ui): implement non-outlined and inline style notification component

### PR DESCRIPTION
**Pull request by realdreamer**
- Introduced the variant as `standard` and `outlined`. But I'm open for discussion here whether we wanna go with outline as a boolean prop. 
- Added inline boolean prop to make the inline notification component for medium and larger screens. 

If we're agreeing to the implementation then I'll update all app/libs which are already consuming notification component

~TODO:~
- [x] Update apps/libs already consume notification component.

### CODE REVIEW

File: packages/libs/react-ui/src/components/Notification/Notification.css.ts [hunk 1]
```diff
@@ -44,6 +43,29 @@ export const expandVariants = styleVariants({
...
+    sprinkles({
+      display: 'flex',
+      flexDirection: 'column',
+    }),
+    {
+      '@media': {
+        [`screen and ${breakpoints.md}`]: {
+          alignItems: 'flex-start',
+          flexDirection: 'row',
+        },
```
Comment (eileenmguo) [inline packages/libs/react-ui/src/components/Notification/Notification.css.ts:62]: I believe you can put this in sprinkles

**Review by eileenmguo — COMMENTED** [review]
I think using the variant makes sense for the actual style. Do you think expanded and inline could be a variant as well since it seems like they can't both be applied?

File: packages/libs/react-ui/src/components/Notification/Notification.css.ts (hunk 1 above)
Comment (eileenmguo) [reply packages/libs/react-ui/src/components/Notification/Notification.css.ts:62]: you can use this syntax ```
```suggestion
sprinkles({
alignItems: {
md: "flex-start",
}), 
flexDirection: {
md: "row"
}
})
```

File: packages/libs/react-ui/src/components/Notification/Notification.stories.tsx [hunk 2]
```diff
@@ -21,6 +21,12 @@ const meta: Meta<
     },
   },
   argTypes: {
+    variant: {
+      options: ['standard', 'outlined'],
```
Comment (timoheddes) [inline packages/libs/react-ui/src/components/Notification/Notification.stories.tsx:25]: Can be `options: Object.keys(displayVariants) as (keyof typeof displayVariants)[],`

**Comment by realdreamer** [comment]
> Do you think expanded and inline could be a variant as well since it seems like they can't both be applied?

It can applied. expanded makes it full width. the `inline` makes the content inside of the `Notification` block to be next to each other. 

image

File: packages/libs/react-ui/src/components/Notification/Notification.css.ts (hunk 1 above)
Comment (realdreamer) [reply packages/libs/react-ui/src/components/Notification/Notification.css.ts:62]: Thank you! I'll try that syntax :)

**Review by timoheddes — APPROVED** [review]

**Review by eileenmguo — APPROVED** [review]

**Pull request merged**